pkg/testcase/snapshotwebhook.go

```diff
@@ -0,0 +1,36 @@
+package testcase
+
+import (
+	"regexp"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestSnapshotWebhook(applyWorkload, deleteWorkload bool) {
+	var workloadErr error
+	if applyWorkload {
+		workloadErr = shared.ManageWorkload("apply", "snapshot-webhook.yaml")
+		Expect(workloadErr).NotTo(HaveOccurred(), "Snapshot Webhook manifest not deployed")
+	}
+
+	cmd := "kubectl get endpoints -n kube-system rke2-snapshot-validation-webhook --kubeconfig=" +
+		shared.KubeConfigFile
+
+	ipPattern := `\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}:8443`
```

Comment:
Let's add two assertions 1. ep should not be empty 2. workload should return somthing like this `
Error from server: error when creating "STDIN": admission webhook "rke2-snapshot-validation-webhook.csi.kubernetes.io" denied the request: Spec.VolumeSnapshotClassName must not be the empty string`

Reply:
1. ep should not be empty 

ip should not be empty ?  @ShylajaDevadiga

Reply:
yes. here is an example 
```
kubectl get endpoints -n kube-system rke2-snapshot-validation-webhook
NAME                               ENDPOINTS   AGE
rke2-snapshot-validation-webhook               107m
```
Expected output
```
kubectl get endpoints -n kube-system rke2-snapshot-validation-webhook
NAME                               ENDPOINTS           AGE
rke2-snapshot-validation-webhook   10.42.96.137:8443   77m
```